base/src/commonTest/kotlin/utils/TemperatureTest.kt

```diff
@@ -0,0 +1,70 @@
+/*
+ Copyright 2020 Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+
+package utils
+
+import com.splendo.kaluga.base.utils.Locale
+import com.splendo.kaluga.base.utils.TemperatureMeasurement
+import com.splendo.kaluga.base.utils.TemperatureUnit
+import kotlin.math.abs
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.asserter
+
+private fun messagePrefix(message: String?) = if (message == null) "" else "$message. "
+
+private fun assertEquals(expected: Double, actual: Double?, epsilon: Double, message: String? = null) {
+    asserter.assertNotNull(null, actual)
+    asserter.assertTrue(
+        { messagePrefix(message) + "Expected <$expected>, actual <$actual>, should differ no more than <$epsilon>." },
+        abs(expected - actual!!) <= epsilon
+    )
+}
+
+class TemperatureTest {
+
+    @Test
+    fun testTemperatureUnitSymbol() {
+        val unitC = TemperatureUnit.CELSIUS
+        assertEquals("°C", unitC.symbol)
+
+        val unitF = TemperatureUnit.FAHRENHEIT
+        assertEquals("°F", unitF.symbol)
```

Comment:
I'm not against non-ascii characters in source files, but it's good practices when testing to test against the explicitly defined unicode code point (I think Kotlin supports `"\u2202F"`), in case some file gets parsed using the wrong encoding (since both main and test sources are compiled this test could pass even if the char is wrongly interpreted).